spec/system/claims/schools/users/copy_view_users_spec.rb

```diff
@@ -1,68 +1,68 @@
 require "rails_helper"
 
 RSpec.describe "Invite and view a users details copy", service: :claims, type: :system do
-  scenario "I sign in as a lead mentor user and invite a user to a school and view that users details" do
+  scenario do
     given_a_school_exists_with_claims_users
-    and_i_am_signed_in_as_a_lead_mentor_user
+    and_i_am_signed_in_as_a_user
 
     when_i_navigate_to_the_claim_school_users_page
-    # then_i_should_see_the_page_title
-    and_i_see_the_claim_page_containing_a_list_of_users_ordered_by_full_name
+    then_i_see_the_claim_users_page
     and_i_select_add_user
     then_i_fill_in_the_new_user_details
     and_i_select_continue
-    then_i_see_the_confirm_user_page
+    then_i_see_the_confirm_user_details_page
     and_i_select_confirm_and_add_user
-    then_i_see_the_new_user_ordered_in_the_users_list
+    then_i_see_the_claim_users_page_with_new_user
```

Comment:
```suggestion
    when_i_select_add_user
    then_i_see_the_add_user_page
    
    when_i_fill_in_the_new_user_details
    and_i_select_continue
    then_i_see_the_confirm_user_details_page
    
    when_i_select_confirm_and_add_user
    then_i_see_the_claim_users_page_with_new_user
```

Could we restructure these methods to more accurately represent the user's journey?